src/App.jsx

```diff
@@ -8,10 +8,11 @@ function App() {
 
   const [todoList, setTodoList] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
+  const [isAscending, setIsAscending] = useState(true);
 
   useEffect(() => {
-    fetchData()
-  }, []);
+    fetchData();
+  }, [isAscending]);
```

Comment:
This won't work as intended. When the page loads, `fetchData` won't be called anymore since there is an explicit dependency in this useEffect, meaning initially no data will be loaded.

Also, you'll be unnecessarily fetching data from the API when you just want to switch the sorting (you can just do that on the client directly).

What I would suggest is 1) revert this useEffect to he previous version with an empty dependency array, then 2) create a separate useEffect that takes `isAscending` as a dependency but just does a local sort of the `todoList` array.